internal/commands/platform/export_internal.go

```diff
@@ -125,16 +122,16 @@ func RunInternalExport(ctx context.Context, commandVersion string) (err error) {
 
 func initPingFederateServices(ctx context.Context, pingcliVersion string) (err error) {
 	if ctx == nil {
-		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: ErrNilContext}
+		return &errs.PingCLIError{Prefix: exportErrorPrefix, Err: auth_internal.ErrPingFederateContextNil}
```

Comment:
I'm not sure if it's typical to import error constants from other packages - might be better to just make a new one for this package. Would be worth seeing if this is done elsewhere.